dom.go

```diff
@@ -709,29 +712,110 @@ func Text(text string, m ...MarkupOrChild) *HTML {
 }
 
 // Rerender causes the body of the given component (i.e. the HTML returned by
-// the Component's Render method) to be re-rendered.
+// the component's Render method) to be re-rendered.
```

Comment:
This should be left as-is, it is intentionally written `Component` referencing the `Component` interface, same as how `Render` is capitalized here.

Reply:
I only changed this for consistency - it was the only capitalisation in this comment.

Reply:
Oh got it. Maybe change it the other way (`component` -> `Component`) ?